src/pages/docs/projects/project-triggers/external-feed-triggers.md

```diff
@@ -7,69 +7,76 @@ description: External feed triggers allow you to automatically create a new rele
 navOrder: 12
 ---
 
-By configuring your Octopus project with container dependencies, you can now create triggers that watch those repositories for new packages pushed by your build tool. Based on tags and version rules, triggers detect if an image appears that is later than the image used in your previous release. Octopus then automatically creates a new release with all the latest container images or Helm chart dependencies. 
+By configuring your Octopus project with container dependencies, you can now create triggers that watch those repositories for new packages pushed by your build tool. Based on tags and version rules, triggers detect if an image appears that is later than the image used in your previous release. Octopus then automatically creates a new release with all the latest container images or Helm Chart dependencies. 
 
 Your existing [lifecycles](/docs/releases/lifecycles/) will then promote that release through your [environments](/docs/infrastructure/environments) or [tenants](/docs/tenants), just like it does currently. If your lifecycle uses automatic release progression, then you've just set up a Continuous Delivery pipeline without explicitly letting Octopus know about your application changes! 
 
-The details of these container images and Helm charts are already known in Octopus. This means we can use the registry locations, image names, chart names, and credentials to do this monitoring, without adding or maintaining this information anywhere else.
+The details of these container images and Helm Charts are already known in Octopus. This means we can use the registry locations, image names, chart names, and credentials to do this monitoring, without adding or maintaining this information anywhere else.
 
 ## Getting started {#ExternalFeedTriggers-GettingStarted}
 
-Navigate to your project’s triggers page by selecting Projects and clicking on the project you are working with. Click **Triggers** option on the left, under **Deployments**.  You are now presented with an empty triggers list.  
+Navigate to your project’s triggers page by selecting Projects and clicking on the project you are working with. Click **Triggers** option on the left, under **Deployments**. You are now presented with an empty triggers list. 
 
 Click the **ADD TRIGGER** on the right-hand side of the page, and select **External feed**
 
 :::figure
 ![Project triggers list](/docs/projects/project-triggers/images/add-trigger-popup.png)
 :::
 
-Enter a name and description for your trigger.  The name should be short, memorable, and unique. Example: Create a release when nginx is updated in Docker.
+Enter a name and description for your trigger. The name should be short, memorable, and unique. Example: Create a release when nginx is updated in Docker.
```

Comment:
Could we make the example name a bit shorter? Something like _Nginx Docker Update_.